src/lib/environment/mod.rs

```diff
@@ -533,6 +533,11 @@ pub(crate) fn setup_cwd(cwd: Option<&str>) {
                 "CARGO_MAKE_CARGO_HOME",
                 &home::cargo_home_with_cwd(directory_path).ok(),
             );
+            envmnt::set_or_remove(
+                "CARGO_MAKE_CRATE_TARGET_TRIPLE",
+                &crateinfo::crate_target_triple(env::current_dir().unwrap())
```

Comment:
not a fan of the unwrap as it may panic. can we change to unwrap_or(".")? would it work?

Reply:
I am not sure 😕.
Maybe we should just fail if we can't aquire the current directory?

Reply:
Alright, I changed it to fail if we can't get the current directory.